readme.md

```diff
@@ -13,6 +13,145 @@ The project and program contain the following elements.
   - **Status**:  The project is currently focussed on preparing t-route for application NWM 3.0.
   - **Demos**: The `/test` directory contains a canned t-route demonstration of routing on the Lower Colorado River, TX for a standard NWM Analsis and Assimilation simulation. A suite of operationally useful capabilities are demonstrated, including streamflow data assimilation, diffusive wave routing along the Lower Colorado mainstem, waterbody simulation, and RFC reservoir forecasting. 
 
+##
+##
+## INSTRUCTIONS TO USE ON WSL (Windows Subsystem for Linux), SPECIFICALLY ON UBUNTU 20.04 (Long Term Stable distribution)
+## ADDED 09-Aug-2023 (legacy instructions see below)
+##
+##
+
+## Install Ubuntu on WSL2 (any Windows 10/11 PC; tested on Windows 11)
+
+Open Windows PowerShell, command:
+
+>> wsl --install Ubuntu-20.04
+
+The Linux distribution will be installed and tracked by a progress bar in the Power Shell window. Once done, you will be prompted to enter a Linux username and password - please do so. The distribution is automatically, and you will be logged into a Linux terminal. 
+
+
+For verification, you can open a new PowerShell, and verify creation of the distribution using the following command:
+
+>> wsl -l -v
+
+, which should result in something like the following (where the line with Ubuntu-20.04 should have been added; other lines are optional):
+
+  NAME             STATE           VERSION
+* rhel79           Stopped         2
+  Ubuntu-20.04     Running         2
+  Ubuntu_v2        Stopped         2
+  Ubuntu           Stopped         2
+  Debian           Running         2
+  Ubuntu_Server    Stopped         2
+
+
+##
+## In Ubuntu: perform updates
+##
+
+>> sudo apt update
+
+[OPTIONAL: If there are strange error messages about updates not being valid yet, that's due to the system clock, and can be fixed by synchronizing the hardware clock:
+>> sudo hwclock --hctosys ]
+
+>> sudo apt -y upgrade
+
+
+##
+## Install Anaconda (in Ubuntu)
+##
+
+Check the archive in  https://repo.continuum.io/archive for available releases. Comment: a stable older version is given as an example here.
+
+Download the distribution:
+
+>> wget https://repo.continuum.io/archive/Anaconda3-2022.05-Linux-x86_64.sh
+
+
+Install Anaconda - Review and confirm license agreement. As installation directory, installed in home directory /home/jz/anaconda3 instead of accepting default installation directory (/root/anaconda3). When done,  agree to running conda init.
+
+>> sudo bash ./Anaconda3-2023.07-1-Linux-x86_64.sh
```

Comment:
This line should be "sudo bash ./Anaconda3-2022.05-Linux-x86_64.sh"

I think if line 67 changes  in a way to get the latest stable version (without specifying the version) that would be better